parkhangs-expressapp/models/index.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
```

Comment:
should we split this file into different files? one for each model? Since in the future we might add more models and this file will get really big

Reply:
I split them up in the new commit